perf/liberty/scripts/bin/common_utils.sh

```diff
@@ -719,8 +719,8 @@ calculateFootprint()
       # now find the footprint figure from the awk output
       local FOOTPRINT=`awk '/JTC footprint/ { print $8 }' footprintAwk.tmp`
       # TODO: Once verified, remove variables not used
-      #local free=`vmstat -l | sed -e 's/  */ /g' |tr ' ' '\n' | tail -n 1 | tail -c 10`
-      #active=`vmstat -l | sed -e 's/  */ /g' |tr ' ' '\n' | tail -n 2 | head -n 1| tail -c 10`
+      #local free=`vmstat -l | sed -e 's/  */ /g' |tr ' ' '\n' | tail -1 | tail -c 10`
```

Comment:
Can this commented out code be removed now?

Reply:
Quite probably, unless it's useful as a reference to someone.
Looks like it was put there by @piyush286 a [couple of years ago](https://github.com/AdoptOpenJDK/openjdk-tests/pull/1128)

Reply:
We can remove it. Thanks Stewart!

Reply:
@piyush286 Are those comments still required or is it ok if I delete them now?

Reply:
@sxa We can remove those commented out lines.